fix(curriculum): description improvement case converter program (#54051)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/663b16e62fee463b4caf46e9.md

---
id: 663b16e62fee463b4caf46e9
title: Step 18
challengeType: 20
dashedName: step-18
---

# --description--

After joining the elements of the list `snake_cased_char_list`, you will need to remove any leading or trailing underscores from the resulting string. For this, use the `strip` method with the underscore character `_` as an argument.

Method calls can be chained together, which means that the result of one method call can be used as the object for another method call.

```py
words_list = ['hello', 'world', 'this', 'is', 'chained', 'methods']
result = ' '.join(words_list).upper()
```

> In the example above, the `join` method is called on the list `words_list`, and the result is used as the object for the `upper` method call.

Reviewer: ```suggestion
In the example above, the `.upper()` method is chained to `' '.join(words_list)`, therefore `.upper()` is called on the result of the `.join()` call.
```

I would reword it a bit. Not necessarily like that.
